tests/webroot/visit-counter.php

```diff
@@ -6,6 +6,10 @@
 
 require_once __DIR__.'/../../vendor/autoload.php';
 
+if (isset($_GET['set_time_limit'])) {
```

Comment:
Can you change the query param name to `no_time_limit` or `with_no_time_limit`? It might be a bit more clear when rereading a few months down the road.